d2/src/main/java/com/linkedin/d2/balancer/D2ClientBuilder.java

```diff
@@ -479,6 +480,12 @@ public D2ClientBuilder setD2JmxManagerPrefix(String d2JmxManagerPrefix)
     return this;
   }
 
+  public D2ClientBuilder setEnableRelativeLoadBalancer(boolean enableRelativeLoadBalancer)
+  {
+    _config.enableRelativeLoadBalancer = enableRelativeLoadBalancer;
```

Comment:
Yes it will be a problem if server only has "RELATIVE" and client set it to enableRelativeLoadBalancer to false. Initially I want server to always set both in the strategy list before we turn on the enableRelativeLoadBalancer to true by default for everyone. So before full rollout, we will enable relative on client as the default behavior.

But we still do not have fully control. One thing I can do is to use "degrader" as the backup, but if server really removed the degrader support, we may have to read global default degrader config for that case. Let me know what you think.

Reply:
I understand we need this for testing purpose. What I'm expecting is that we remove it before fully roll out.

If you look at r2d2 configuration, there are very few properties are overridable from client side. Even if one does, it still needs permissions from the server side. Having this flag makes it implicit if relativeLB is used or not when you investigate a problem. It's probably better have it only enable it from d2 config directly.

Reply:
Since the default client-side behavior after rollout would enable the relative strategy, then if a client explicitly calls `setEnableRelativeLoadBalancer` to disable the relative feature, they probably should have a good reason to do so. If we're adding this client-side tuner, is it more reasonable to use `degrader` as a backup in this case?

Reply:
Makes sense, to summarize:
1. We will only use enableRelativeLoadBalancer for piloting services, after pilot we will turn it on for everyone and rely on server side control.
2. In the short-term, if anyone set the enableRelativeLoadBalancer to false, I will add the backup option.

Reply:
Sounds great.